tools/capacity-planner-sheets-extension/README.md

```diff
@@ -0,0 +1,51 @@
+# Capacity Planner Sheets Extension
+
+This tool is a Google Sheets extension for capacity planning on Google Cloud. 
+This tool extracts peak resource usage values and corresponding timestamps for a given Google Cloud project, time range and timezone.
+
+
+## Setup
```

Comment:
As long as you get the desired results, the .xlsx upload as a template approach should work okay. I understand that making a google sheet as a template public might not be a good idea. Another possibility (potentially future) would be have an init function step (implemented in the same app script) that initialises a given new sheet with the placeholders needed before usage. But this would involve adding more code as well, so it's really up to you and whatever works for your users.

Reply:
Thanks for the advice @aymanfarhat!  I unfortunately don't have additional cycles to spend on this right now, so I'll leave it like this for now. 

If I end up getting the time/budget to do more work on this I'd definitely like to add an auto-init function.

Reply:
ACK 👍